orangecontrib/imageanalytics/image_embedder.py

```diff
@@ -193,11 +196,21 @@ def _send_to_server(self, file_paths, image_processed_callback):
         )
 
     def _load_image_or_none(self, file_path):
-        try:
-            image = open_image(file_path)
-        except IOError:
-            log.warning("Image skipped (invalid file path)", exc_info=True)
-            return None
+	parsed_url = urlparse(file_path)
```

Comment:
Please check your indentation. Some indents don't work and thus the code fails. You need to go 4 spaces (or 1 tab) in under a function. If you are using a smart IDE (e.g. PyCharm), it should notify you of the problematic indent and also warn you about breaking PEP8 (going over 80 character limit).